packages/document/main-doc/docs/en/blog/index.md

```diff
@@ -4,5 +4,70 @@ sidebar_position: 1
 
```

Comment:
The `en/blog` folder has been moved to `en/community/blog`, see [/en/community/blog](https://github.com/web-infra-dev/modern.js/tree/main/packages/document/main-doc/docs/en/community/blog).

Please pull the latest code in the main branch~

Reply:
sure!